apps/pipelines/models.py

```diff
@@ -504,6 +504,12 @@ class PipelineChatHistoryTypes(models.TextChoices):
     NONE = "none", "No History"
 
 
+class PipelineChatHistoryModes(models.TextChoices):
+    SUMMARIZE = "summarize", "Summarize"
+    TRUNCATE_TOKENS = "truncate tokens", "Truncate Tokens"
+    MAX_HISTORY_LENGTH = "max history length", "Max History Length"
```

Comment:
nit: values should be slugs
```suggestion
    SUMMARIZE = "summarize", "Summarize"
    TRUNCATE_TOKENS = "truncate_tokens", "Truncate Tokens"
    MAX_HISTORY_LENGTH = "max_history_length", "Max History Length"
```